wire-runtime-swift/src/main/swift/ProtoCodable/ProtoIntCodable+Implementations.swift

```diff
@@ -48,7 +48,12 @@ extension Int32: ProtoIntCodable, ProtoDefaultedValue {
             writer.writeVarint(zigZagEncoded())
         case .variable:
             // int32 fields
-            writer.writeVarint(UInt32(bitPattern: self))
+            if (self >= 0) {
+                writer.writeVarint(UInt32(bitPattern: self))
+            } else {
+                // Must sign-extend.
+                writer.writeVarint(UInt64(bitPattern: Int64(self)))
```

Comment:
This is the breaking change in encoding in which we now correctly encode negative values as unsigned int 64 as per protobuf spec.